simbev/mid_timeseries.py

```diff
@@ -0,0 +1,105 @@
+import pandas as pd
+# import numpy as np
+import math
+import datetime
+from pathlib import Path
+
+
+# set cutoff dates, jeweils erster Monat der nächsten Jahreszeit
+cutoff_dates = (3, 6, 9, 12)
```

Comment:
better add hard coded variables into the config files and load them into the script